files/source/galaxy_workflows.yml

```diff
@@ -0,0 +1,46 @@
+workflows:
+  # this list of workflows and their annotations are a wip and will be updated as i learn more
+  # about their status and the tools used, etc.
+  - trs_id: "#workflow/github.com/iwc-workflows/chipseq-pe/main"
+    type: "REGULATION"
+    description: "Workflows for the analysis of ChIP-seq, ATAC-Seq, and beyond."
+    ploidy: "any"
+
+  - trs_id: "#workflow/github.com/iwc-workflows/chipseq-sr/main"
+    type: "REGULATION"
+    description: "Workflows for the analysis of ChIP-seq, ATAC-Seq, and beyond."
+    ploidy: "any"
+
+  - trs_id: "#workflow/github.com/iwc-workflows/rnaseq-pe/main"
+    type: "TRANSCRIPTOMICS"
+    description: "Analyze bulk or single-cell RNA seq data using a variety of approaches."
+    ploidy: "any"
+
+  - trs_id: "#workflow/github.com/iwc-workflows/rnaseq-sr/main"
+    type: "TRANSCRIPTOMICS"
+    description: "Analyze bulk or single-cell RNA seq data using a variety of approaches."
+    ploidy: "any"
+
+  - trs_id: "#workflow/github.com/iwc-workflows/haploid-variant-calling-wgs-pe/main"
+    type: "VARIANT_CALLING"
+    description: "Identify nucleotide polymorphisms and short indels from Illumina and Element data."
+    ploidy: "haploid"
+
+  # im starting to wonder looking at this if the default for ploidy shouldnt be any
+  # maybe it could be multivalued [haploid, diploid].. might not matter depending on ui
+  - trs_id: "#workflow/github.com/iwc-workflows/generic-variant-calling-wgs-pe/main"
+    type: "VARIANT_CALLING"
+    description: "Identify nucleotide polymorphisms and short indels from Illumina and Element data."
+    ploidy: "any"
+
+  - trs_id: "#workflow/github.com/iwc-workflows/assembly-with-flye/main"
+    type: "ASSEMBLY"
+    description: "Best practices for assembly of prokaryotic and eukaryotic genomes sequenced with a variety of technologies."
+    ploidy: "any"
+
+  # idk yet what makes this specific to bacteria, or if it can be easily generalized
```

Comment:
Uses https://github.com/tseemann/shovill

Reply:
Thanks. maybe it's enough to annotate as haploid then..